x/auth/client/cli/tx_multisign.go

```diff
@@ -182,8 +183,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 
 		outputDoc, _ := cmd.Flags().GetString(flags.FlagOutputDocument)
```

Comment:
Can we clean up the following and use `closeFunc` like for batch multisign: https://github.com/cosmos/cosmos-sdk/blob/33051ceb398d1279c8897add74314111dd8684a2/x/auth/client/cli/tx_multisign.go#L296?

I am not sure it even works currently because we do not set the output of the command.